src/test/test.js

```diff
@@ -1,19 +1,31 @@
-import chai from 'chai';
+import chai, { expect } from 'chai';
 import chaiHttp from 'chai-http';
 
 import app from '../app';
 
-const { expect } = pkg;
 chai.use(chaiHttp);
 
 describe('Server!', () => {
-  it(' the api testing', (done) => {
+  it('the api testing', (done) => {
     chai
       .request(app)
       .get('/')
       .end((err, res) => {
         expect(res).to.have.status(200);
-        expect(res.body.message).to.equals('Welcome to phantom, a platform to facilitate the transportation mode in town!.');
+        expect(res.body.message).to.equals('Welcome to phantom!.');
+        done();
+      });
+  });
+});
+
+describe('Sign in route', () => {
+  it('check user info', (done) => {
+    chai
+      .request(app)
+      .post('/v1/auth/login')
+      .end((err, res) => {
+        expect(res).to.have.status(401);
+        expect(res.body.message).to.equals('wrong email or password');
         done();
       });
   });
```

Comment:
Add (at least) one more test for the case where a user provides the right email and password.